clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -113,6 +115,37 @@ class GoogleServerGroup {
           null
     }
 
+    /**
+     * @return The load balancing policy containing the
+     * capacity metrics and named ports this server group is configured with for all L7 backends.
+     */
+    GoogleHttpLoadBalancingPolicy getLoadBalancingPolicy() {
+      def loadBalancingPolicy = null
+      def loadBalancers = GoogleServerGroup.this.loadBalancers
+      loadBalancers.each { GoogleLoadBalancerView view ->
+        if (GoogleLoadBalancerType.valueOf(view.loadBalancerType) == GoogleLoadBalancerType.HTTP) {
+          GoogleHttpLoadBalancer.View httpView = view as GoogleHttpLoadBalancer.View
+          def backendServices = Utils.getBackendServicesFromHttpLoadBalancerView(httpView)
+          backendServices.each { GoogleBackendService backendService ->
+            def foundBackends = backendService.backends.findAll { GoogleLoadBalancedBackend backend ->
+              GCEUtil.getLocalName(backend.serverGroupUrl) == GoogleServerGroup.this.name
+            }
+            // Note: we configure the same load balancing policy for all backends on a create/clone,
+            // so we can just return the policy from the first one we find.
+            if (foundBackends) {
+              def foundBackend = foundBackends?.first()
+              loadBalancingPolicy = foundBackend?.policy
+              if (loadBalancingPolicy) {
+                loadBalancingPolicy.listeningPort = GoogleServerGroup.this.namedPorts.get(GoogleHttpLoadBalancingPolicy.HTTP_PORT_NAME)
+              }
+              return loadBalancingPolicy
```

Comment:
Hmm the outer loop keeps looping though, and `return`ing an ignored value is a bit odd.